linux/utils/install_uninstall_package.yml

```diff
@@ -105,31 +105,23 @@
         allow_downgrade: yes
         state: "{{ package_state }}"
       delegate_to: "{{ vm_guest_ip }}"
-      when: guest_os_ansible_distribution_major_ver | int <= 7
+      when: guest_os_ansible_pkg_mgr | lower == "yum"
 
     - name: "{{ local_task_name }} on {{ guest_os_ansible_distribution }} {{ guest_os_ansible_distribution_ver }}"
       dnf:
         name: "{{ package_name }}"
         allow_downgrade: yes
         state: "{{ package_state }}"
       delegate_to: "{{ vm_guest_ip }}"
-      when: guest_os_ansible_distribution_major_ver | int >= 8
-  when: guest_os_ansible_distribution in ["RedHat", "CentOS", "OracleLinux"]
-
-- name: "{{ local_task_name }} on {{ guest_os_ansible_distribution }} {{ guest_os_ansible_distribution_ver }}"
-  yum:
-    name: "{{ package_name }}"
-    state: "{{ package_state }}"
-    update_cache: "{{ update_cache | default(True) }}"
```

Comment:
Is "update_cache" parameter still needed here?

Reply:
Yeah, but "update_cache" parameter is removed in this task `install_uninstall_package.yml`.